Missing source files in Visual Studio 2015 project

Undefined references are encountered when building with MSVC 2015:

```
arena.obj : error LNK2001: unresolved external symbol test_hooks_arena_new_hook [C:\w\jemalloc\msvc\pro
jects\vc2015\jemalloc\jemalloc.vcxproj]
malloc_io.obj : error LNK2001: unresolved external symbol test_hooks_libc_hook [C:\w\jemalloc\msvc\proj
ects\vc2015\jemalloc\jemalloc.vcxproj]
pages.obj : error LNK2001: unresolved external symbol test_hooks_libc_hook [C:\w\jemalloc\msvc\projects
\vc2015\jemalloc\jemalloc.vcxproj]
prof.obj : error LNK2001: unresolved external symbol test_hooks_libc_hook [C:\w\jemalloc\msvc\projects\
vc2015\jemalloc\jemalloc.vcxproj]
```

Potential solution:

```diff
diff --git a/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj b/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj
index 228e8be0..3376ad4f 100644
--- a/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj
+++ b/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj
@@ -64,4 +64,5 @@
     <ClCompile Include="..\..\..\..\src\sz.c" />
     <ClCompile Include="..\..\..\..\src\tcache.c" />
+    <ClCompile Include="..\..\..\..\src\test_hooks.c" />
     <ClCompile Include="..\..\..\..\src\ticker.c" />
     <ClCompile Include="..\..\..\..\src\tsd.c" />
diff --git a/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj.filters b/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj.filters
index d839515b..ed5fcc9d 100644
--- a/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj.filters
+++ b/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj.filters
@@ -83,4 +83,7 @@
       <Filter>Source Files</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\..\..\src\test_hooks.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
     <ClCompile Include="..\..\..\..\src\ticker.c">
       <Filter>Source Files</Filter>
```

Both the MSVC 2015 and 2017 projects broke as part of c7a87e0e0bd02cf278760f3c22615d3129dc1ae2. The MSVC 2017 project was fixed as part of ce5c073fe5017e802d1e272a9e057f7b631da345 (but not the MSVC 2015 project).

--- a/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj
+++ b/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj
@@ -65,6 +65,7 @@
     <ClCompile Include="..\..\..\..\src\stats.c" />
     <ClCompile Include="..\..\..\..\src\sz.c" />
     <ClCompile Include="..\..\..\..\src\tcache.c" />
+    <ClCompile Include="..\..\..\..\src\test_hooks.c" />
     <ClCompile Include="..\..\..\..\src\ticker.c" />
     <ClCompile Include="..\..\..\..\src\tsd.c" />
     <ClCompile Include="..\..\..\..\src\witness.c" />

--- a/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj.filters
+++ b/msvc/projects/vc2015/jemalloc/jemalloc.vcxproj.filters
@@ -88,6 +88,9 @@
     <ClCompile Include="..\..\..\..\src\tcache.c">
       <Filter>Source Files</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\..\..\src\test_hooks.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
     <ClCompile Include="..\..\..\..\src\ticker.c">
       <Filter>Source Files</Filter>
     </ClCompile>

